weave/flow/scorer.py

```diff
@@ -35,6 +37,17 @@ def summarize(self, score_rows: list) -> Optional[dict]:
         return auto_summarize(score_rows)
 
 
+class BuiltInScorer(Scorer):
```

Comment:
This seems odd. Searching the current code base, I see this pattern exists elsewhere, but it certainly seems odd to me that we need to explicitly do this. Adding an intermediate class definition with a general-purpose deserializer?? Why is this not just a class method for on the `Object` class? Maybe question for @andrewtruong?